op-node/flags/flags.go

```diff
@@ -263,6 +263,7 @@ var (
 		Usage:    "Path to the snapshot log file",
```

Comment:
Is it worth adding to the usage that its a noop?

Reply:
I suppose if it never shows up then no

Reply:
Good idea, clarified the wording